packages/ra-ui-materialui/src/button/BulkDeleteWithConfirmButton.tsx

```diff
@@ -37,7 +37,7 @@ export const BulkDeleteWithConfirmButton = (
         ...rest
     } = props;
     const { meta: mutationMeta, ...otherMutationOptions } = mutationOptions;
-    const { selectedIds, onUnselectItems } = useListContext(props);
+    const { selectedIds, onUnselectItems } = useListContext();
```

Comment:
Shouldn't it call `useListContextWithProps`? It seems we loose the ability to override through props as before. Same for all the other UI components

Reply:
you're right, but I don't see any use case for overriding the selectedIds on this one.